### ISSUE TITLE
Provide outputEntity selector in marker config panel(s)

### ISSUE THREAD

**Issue by bobular** [VEuPathDB/web-monorepo#304]
For Donut and Chart markers I think we'd want both selectors at the top (see updated spec for #190)
* Variable selector
* Output entity selector

For future bubble markers it would be the other way round, right?
* Entity selector (for bubble size)
* optional Variable selector (for color overlay)

Obviously "Output entity" is too technical. "Show counts for"?

Cost is 1-2 days. All front end. Data service already ready to party.

**Comment by bobular** [comment]
I'm thinking we should close this as "won't fix".  

Assuming there's a bubble-friendly categorical variable for each entity, the default bubble mode will soon (after #416) show simple entity counts until the numerator/denominator is configured, at least.

We could come back to it, but let's have users play with it first?

**Comment by bobular** [comment]
Closing. Let's reopen or make a new ticket if this comes up in real life use cases.
